panimg/models.py

```diff
@@ -39,6 +43,70 @@ class EyeChoice(str, Enum):
     NOT_APPLICABLE = "NA"
 
 
+class PatientSex(str, Enum):
+    MALE = "M"
+    FEMALE = "F"
+    OTHER = "O"
+
+
+DICOM_VR_TO_VALIDATION_REGEXP = {
+    "AS": re.compile(r"^\d{3}[DWMY]$"),
+    "CS(PatientSex)": re.compile(
+        f"^[{''.join([s.value for s in PatientSex])}]$"
```

Comment:
Isn't this being done in the enum validation?

Reply:
Just checked and indeed pydantic takes care of it. Nice!